### PR TITLE
Fixes to webapp

### PR DESCRIPTION
**Pull request by brusMX**
Fixes most of the reported bugs

### CODE REVIEW

File: webapp/src/components/home/homeContainer.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default class HomeContainer extends React.Component {
   componentDidMount() {
     //Get Keg status
     var myHeaders = new Headers();
+    var actAmount = 15;
```
Comment (jamesbak) [inline webapp/src/components/home/homeContainer.js]: Is this variable ever used?

File: webapp/src/components/home/homeContainer.js [hunk 2]
```diff
@@ -50,7 +51,7 @@ export default class HomeContainer extends React.Component {
       }
       });
 
-    fetch(webAppConfig.api.url+'/activity',myInit)
+    fetch(webAppConfig.api.url + '/activity?count=20',myInit)
```
Comment (jamesbak) [inline webapp/src/components/home/homeContainer.js]: I don't think this completely resolves the issue. The fact that the panel is not responsive to the size of the viewport is the bigger issue that this needs to address.

Comment (nzthiago) [inline webapp/src/components/home/homeContainer.js]: @jamesbak - do you want a scrollable view for the check ins panel, or should it not have a scroll at all and just show the amount of check ins that fit in the panel?

Comment (jamesbak) [inline webapp/src/components/home/homeContainer.js]: Nothing should be scrollable - it's a non-interactive dashboard. The panel can overflow if there's more activities than space on the viewport. The viewport should size according to the size of the browser window.

**Comment by jamesbak** [comment]
Bruno - all of the tests are failing for this PR. I suspect you need to rebase master into your branch and pushing that.
I can't approve the PR until the tests pass.

**Pull request merged**